src/features/profile/Profile.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button } from '../../common/components';
+import NarrativeList from '../../common/components/NarrativeList/NarrativeList';
+import {
+  useAppDispatch,
+  useAppSelector,
+  usePageTitle,
+} from '../../common/hooks';
+import { authUsername } from '../auth/authSlice';
+import PageNotFound from '../layout/PageNotFound';
+import {
+  cache,
+  clearError,
+  error,
+  loadProfile,
+  selectUsername,
+  usernameSelected,
+} from './profileSlice';
+import classes from './Profile.module.scss';
+
+export const ProfileNarrativesMessage: FC<{
+  realname: string;
+  yours: boolean;
+}> = ({ realname, yours }) => {
+  if (yours) {
+    return <span>This table shows all of your narratives.</span>;
+  }
+  return (
+    <span>
+      This table shows all narratives owned by {realname} which are also
+      accessible to you.
+    </span>
+  );
+};
+
+export const ProfileResume: FC = () => {
+  return <div>Profile Resume</div>;
+};
+
+export const NarrativesView: FC<{ realname: string; yours: boolean }> = ({
+  realname,
+  yours,
+}) => {
+  return (
+    <div className={classes.narratives}>
+      <ProfileNarrativesMessage realname={realname} yours={yours} />
+      <NarrativeList
+        items={[]}
+        itemsRemaining={0}
+        hasMoreItems={false}
+        loading={false}
+        onLoadMoreItems={() => {}} //eslint-disable-line @typescript-eslint/no-empty-function
+        onSelectItem={() => {}} //eslint-disable-line @typescript-eslint/no-empty-function
+        showVersionDropdown
+      />
+    </div>
+  );
+};
+
+export const ProfileInfobox: FC<{ realname: string }> = ({ realname }) => {
+  return <div>Profile Infobox for {realname}.</div>;
+};
+
+export const ProfileView: FC<{ realname: string }> = ({ realname }) => {
+  return (
+    <div className={classes.profile}>
+      <ProfileInfobox realname={realname} />
+      <ProfileResume />
+    </div>
+  );
+};
+
+export interface ProfileParams {
+  narrativesLink: string;
+  pageTitle: string;
+  profileLink: string;
+  realname: string;
+  username: string;
+  viewMine: boolean;
+  viewNarratives: boolean;
+}
+
+export const Profile: FC<ProfileParams> = ({
+  narrativesLink,
+  pageTitle,
+  profileLink,
+  realname,
+  username,
+  viewMine,
+  viewNarratives,
+}) => {
+  usePageTitle(pageTitle);
+  return (
+    <>
+      <nav className={classes['profile-nav']}>
+        <ul className="links">
+          <li>
+            <Link to={profileLink}>Profile</Link>
+          </li>
+          <li>
+            <Link to={narrativesLink}>Narratives</Link>
+          </li>
+        </ul>
+      </nav>
+      <section className={classes['profile-narratives']}>
+        {viewNarratives ? (
+          <NarrativesView realname={realname} yours={viewMine} />
+        ) : (
+          <ProfileView realname={realname} />
+        )}
+        <Button>Edit</Button>
+      </section>
+    </>
+  );
+};
+
+export const ProfileWrapper: FC = () => {
```

Comment:
I like this wrapper that handles all the redux logic, it makes the storybook display really straightforward. I was having trouble doing something like this within a storybook component for one of the navigator items.

Reply:
Thanks to @dauglyon for the suggestion!